### ISSUE TITLE
Nemo is very slow/unresponsive in home directory

### ISSUE THREAD

**Issue by GitHubIsToxic** [linuxmint/nemo-extensions#94]
I've found Nemo to become practically unusable for browsing anywhere in the home directory with the nemo-fileroller extension installed, to the point where double clicking consistently failed because Nemo was having such a hard time being responsive (I was forced to use the arrow/Enter keys to browse folders, open files etc).

Folder contents display is slow (~2 sec until responsive), right click is very slow (~3 sec until context menu shows), file properties extremely slow (~5 sec or more until the dialog shows up). File operations not noticably affected, except rename of course as it requires a folder refresh (~2 sec).
And generally little hangs and stutter all over, skipped clicks/drags/keypresses. Double clicking fails ~50% of the time.

Doesn't matter if the opened folder is empty or filled with hundreds of (image) files.

Browsing outside home directory was somehow not noticably slow (!).

Thunar has always been responsive and snappy on this machine (so it's not a hardware/file system/networking/thumbnailer issue). All issues disappeared after removing nemo-fileroller.

I would suggest not installing/bundling nemo-fileroller until this problem is traced down and fixed.

Nemo 2.0.8
Linux Mint 13 XFCE
using Nemo as desktop manager

Hope that helps!

**Comment by ByteHazard** [comment]
Actually, I'd like to retract the above statement about nemo-fileroller being responsible for this. After rebooting, Nemo is once again very slow and unresponsive. I don't know why removing that package suddenly made it run better.

The problem persists anyway. Nemo is usable outside the home directory (not very fast but usable), but inside home directory the issues above come into play.

How can I profile what is slowing Nemo down?

**Comment by nagash91** [comment]
I had a similar issue here, 2 years after.
When selecting/browsing folders with arrows, nemo was extremely slow (1~2 seconds to select the next folder). Also opening a folder was slow.

I disabled the extension "ChangeColorFolder+NemoPython" and now is fast again.
(Linux Mint 18.1 Serena)

Thanks

**Comment by ByteHazard** [comment]
In hindsight, the unresponsiveness I reported may have been caused by RabbitVCS, although I'm not 100% sure I had that installed when I experienced these problems. Most versions of RabbitVCS continually print warnings to the log file (quickly growing in file size) when browsing outside a .svn or .git directory (i.e. most of the time).
That said, without any plugins installed Nemo is typically still much slower than Thunar on the same machine.
